core/src/main/scala/org/apache/spark/Accumulators.scala

```diff
@@ -47,7 +48,7 @@ import org.apache.spark.util.Utils
  * @tparam T partial data that can be added in
  */
 class Accumulable[R, T] private[spark] (
-    @transient initialValue: R,
+    @(transient @param @field) initialValue: R,
```

Comment:
/cc @rxin since we talked about this before, and he's the one who added `-XfatalWarnings` to the build.